hello-server.ts

```diff
@@ -0,0 +1,13 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
```

Comment:
Importing is how we get stuff from other files to use. Node comes with some tools for us to build http servers that we need to import.
- createServer: A function that creates an HTTP server.
- IncomingMessage: Represents the request sent by a client.
- ServerResponse: Represents the response sent back to the client.